src/objectTranslations/Payment_Services_Configuration__c-fr.objectTranslation

```diff
@@ -2,40 +2,40 @@
 <CustomObjectTranslation xmlns="http://soap.sforce.com/2006/04/metadata">
     <caseValues>
         <plural>false</plural>
-        <value><!-- Payment Services Configuration --></value>
+        <value>Configuration des services de paiement</value>
     </caseValues>
     <caseValues>
         <plural>true</plural>
-        <value><!-- Payment Services Configuration --></value>
+        <value>Configurations des services de paiement</value>
     </caseValues>
     <fields>
-        <label><!-- Is Secret --></label>
+        <label>Est secret</label>
         <name>Is_Secret__c</name>
     </fields>
     <fields>
-        <label><!-- Key --></label>
+        <label>Clé</label>
         <name>Key__c</name>
     </fields>
     <fields>
-        <label><!-- Service Key --></label>
+        <label>Clé de service</label>
         <name>Service_Key__c</name>
     </fields>
     <fields>
-        <label><!-- Service --></label>
+        <label>Service</label>
         <name>Service__c</name>
     </fields>
     <fields>
-        <label><!-- Value --></label>
+        <label>Valeur</label>
         <name>Value__c</name>
     </fields>
-    <gender><!-- Masculine --></gender>
+    <gender>Feminine</gender>
     <layouts>
         <layout>Payment Configuration Layout</layout>
         <sections>
-            <label><!-- Custom Links --></label>
+            <label>Liens personnalisés</label>
             <section>Custom Links</section>
         </sections>
     </layouts>
-    <nameFieldLabel><!-- Payment Services Configuration Name --></nameFieldLabel>
-    <startsWith><!-- Consonant --></startsWith>
+    <nameFieldLabel>Nom de configuration des services de paiement</nameFieldLabel>
```

Comment:
@dmkud This is being truncated down to 40 characters - "Nom de configuration des services de pai". It might be necessary to change the translation to be a max of 40.

Reply:
Fixed